### ISSUE TITLE
Update to the Ghost 0.5 asset helper

### ISSUE THREAD

**Issue by sethlivingston** [martijnrusschen/swayze#1]
```
<link href="/css/styles.css"...
```

should be

```
<link href="{{ asset "css/styles.css" }}"...
```

...and I believe there are other changes needed to bring it in line with Ghost 0.5. (http://dev.ghost.org/themes-multi-user-ready/)

**Comment by martijnrusschen** [comment]
Thanks for mentioning. Will look into it!

**Comment by martijnrusschen** [comment]
Fixed by 8ac052e100ad88fbb958f957c20356ac232f7e16
